packages/astro/src/core/errors/errors-data.ts

```diff
@@ -780,6 +780,26 @@ export const LocalsNotAnObject = {
 	hint: 'If you tried to remove some information from the `locals` object, try to use `delete` or set the property to `undefined`.',
 } satisfies ErrorData;
 
+/**
+ * @docs
+ * @description
+ * Thrown in development mode when middleware throws an error while attempting to loading it.
+ *
+ * For example:
+ * ```ts
+ * import {defineMiddleware} from "astro:middleware";
+ * throw new Error("Error thrown while loading the middleware.")
+ * export const onRequest = defineMiddleware(() => {
+ *   return "string"
+ * });
+ * ```
+ */
+export const MiddlewareCantBeLoaded = {
+	name: 'MiddlewareCantBeLoaded',
+	title: "Can't load the middleware.",
+	message: 'The middleware thrown an error while Astro was trying to loading it.',
```

Comment:
Hi @ematipico ! There's a tiny language error here! Can you please update to whichever feels right to you?

> message: 'The middleware threw an error while Astro was trying to loading it.',